yb-voyager/src/dbzm/schemaRegistry.go

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package dbzm
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/tgtdb"
+)
+
+type DbzSchema struct {
+	Type string `json:"type"`
+	Name string `json:"name"`
+	// Not decoding the rest of the fields for now.
+}
+
+type Schema struct {
+	ColName      string    `json:"name"`
+	Index        int64     `json:"index"`
+	ColDbzSchema DbzSchema `json:"schema"`
+}
+
+type TableSchema struct {
+	Columns []Schema `json:"columns"`
+}
+
+func GetTableSchema(tableName string, exportDir string) (TableSchema, error) {
```

Comment:
Make it a private method of `SchemaRegistry`.

Almost always: return pointer types instead of value types. In this case, return `*TableSchema`. That will allow you to return `nil` TableSchema in the presence of an error. Fix all such occurrences.